modules/datadict/test/TestPlan

```diff
@@ -25,3 +25,5 @@
    [Automation Testing]
 9. Check that the maximum rows displayed dropdown works, also check that the download table as CSV works as well.
    [Automation Testing]
+10. Check tools/data_dictionary_builder.php works. Try changing field names in an instrument before running script and
+    make sure that the corresponding entries in Data Dictionary are updated correctly.
```

Comment:
There's some grammatical errors in this. It should be "Check that the [...] works." It should also be "before running the script".